pkg/cli/cmd/config/view.go

```diff
@@ -49,7 +48,22 @@ The configuration is displayed in YAML format.
 Examples:
   # View current configuration
   tkn-results config view`,
-		PreRunE: func(_ *cobra.Command, _ []string) error {
+		PreRunE: func(cmd *cobra.Command, _ []string) error {
+			// Set up IOStreams using command's output if available, otherwise use os streams
+			var out, errOut io.Writer = os.Stdout, os.Stderr
+			if cmd.OutOrStdout() != nil {
+				out = cmd.OutOrStdout()
+			}
+			if cmd.ErrOrStderr() != nil {
+				errOut = cmd.ErrOrStderr()
+			}
+
+			opts.IOStreams = &genericiooptions.IOStreams{
+				In:     os.Stdin,
+				Out:    out,
+				ErrOut: errOut,
+			}
```

Comment:
Did you encounter an instance where `cmd.OutOrStdout()` or `cmd.ErrOrStderr()` returned `nil`? [Their implementation](https://github.com/spf13/cobra/blob/v1.9.1/command.go#L393) is such that if the cmd has no special out or err writer then they will return `os.Std{out,err}`. 

From what I can tell it should be safe and a bit simpler to do:
```suggestion
			opts.IOStreams = &genericiooptions.IOStreams{
				In:     cmd.InOrStdin(),
				Out:    cmd.OutOrStdout(),
				ErrOut: cmd.ErrOrStderr() ,
			}
```